src/main/java/org/apache/sling/graphql/core/engine/DefaultQueryExecutor.java

```diff
@@ -219,16 +219,25 @@ public ValidationResult validate(@NotNull String query, @NotNull Map<String, Obj
                     }
                 }
                 if (LOGGER.isErrorEnabled()) {
-                    LOGGER.error("Query failed for Resource {}: query={} Errors:{}, schema={}",
-                            queryResource.getPath(), cleanLog.sanitize(query), errors, schemaDef);
+                    final String errorMessage = String.format("Query failed for Resource %s: query=%s Errors:%s",
+                            queryResource.getPath(), cleanLog.sanitize(query), errors);
+                    if (LOGGER.isDebugEnabled()) {
+                        LOGGER.error(String.format("%s, schema=%s", errorMessage, schemaDef));
```

Comment:
Did you verify that the schema is actually logged? It looks like the `schemaDef` variable is not set, looks like a bug in the existing code. Anyway, I'm not sure if logging the schema is useful, I'll discuss in SLING-10503

Reply:
I did verify that the schema is logged but you're right the `schemaDef` variable is not set since this [commit](https://github.com/apache/sling-org-apache-sling-graphql-core/commit/9abddb77904a079a5c3c58de0901ac576fb6b53f#). The logs I had came for the previous version (0.0.9), my bad, but the code still needs to be fixed.